### PR TITLE
Command Line tab in UWP app stores command history

### LINKED ISSUE #91: Add history to command prompt
The Factory Orchestrator UWP app's command prompt would be much improved with a basic command history, so that it is easy to re-run commands.

https://github.com/microsoft/FactoryOrchestrator/blob/main/src/App/ConsolePage.xaml.cs

### PR DESCRIPTION
**Pull request by spaceisfun**
closes #91

The UWP app's command prompt remembers your last 100 commands with this change.

### CODE REVIEW

File: src/App/ConsolePage.xaml.cs [hunk 1]
```diff
@@ -30,6 +30,11 @@ public ConsolePage()
             _outSem = new SemaphoreSlim(1, 1);
             _activeRunSem = new SemaphoreSlim(1, 1);
             _newCmd = false;
+            _cmdHistory = new List<string>(20);
```
Comment (patelp3989) [inline src/App/ConsolePage.xaml.cs:33]: Can we initialize size of the list to MaxCmdHistory .

**Review by patelp3989 — APPROVED** [review]

File: src/App/ConsolePage.xaml.cs (hunk 1 above)
Comment (spaceisfun) [reply src/App/ConsolePage.xaml.cs:33]: my theory was 20 is the likely upper bound for most people so i init to that, but let you save up to 100 (i'd just need to resize the list).

In either way, i think its a very minor (insignificant) memory/perf trade off, and I'd rather err on the side of using less initial memory at the cost of the list resize perf hit instead of defaulting to use more memory.

tldr - likely irrelevent at this scale, but wont fixing to save a bit of memory.
